### DISCUSSION TITLE
Splash screen

### DISCUSSION THREAD

**Discussion by sikyO** [Shabinder/SpotiFlyer#1046] [Ideas]
Could you make it in such a way that the splash screen about donating could go away after donating? It appears every time I try it and it's a bit annoying to say the least.

Thanks.

**Comment by Shabinder** [comment]
> Could you make it in such a way that the splash screen about donating could go away after donating? It appears every time I try it and it's a bit annoying to say the least.
> 
> Thanks.

It shouldnt come everytime.

If you select dismiss it shouldnt come for next 10 times.

> go away after donating?

Not possible as of now, I am rewriting the whole app so this will be fixed in that.
